.idea/mavenProjectSettings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="MavenProjectSettings">
+    <option name="testRunningSettings">
+      <MavenTestRunningSettings>
+        <option name="passArgLine" value="false" />
```

Comment:
Could you please add line
```
        <!-- See https://github.com/apache/incubator-druid/issues/8524 -->
```
Before this line.

Reply:
Or better reference `https://github.com/apache/incubator-druid/pull/8526` (i. e. this PR) because it has extra details.